src/report_workflow/test_report_runner.py

```diff
@@ -48,6 +52,14 @@ def __init__(self, args: ReportArgs, test_manifest: TestManifest) -> None:
 
     def update_data(self) -> dict:
         self.test_run_data["name"] = self.product_name
+        if float(self.schema_version) > 1.0:
```

Comment:
Any situation that we still want to run on 1.0 schema version? 
or could we have a better mechanism to work between previous schema_version vs current? This seems to be hardcoded. I feel like having a boolean flag might be slightly better but still wonder do we really need this.

Reply:
It is for the purpose of BC.
Note for all of the manifest right now we are following this way.
I do think there should be better way to handle the schema_version > 1.0, but it is not easy at the moment. Happy for any advice right here.

Reply:
I think the most case we worry about BC because we are importing the manifest from `.yml` file so we have a different manifest class as `test_report_manifest_1_0.py`. But in this report workflow, we are generating the yml file from data map. Still think it's not necessary to introduce another argument in this case. We could for example make `--release-candidate` an optional argument; if it's provided, we are generating `1.1` version report manifest; else keep creating old version manifest if we would still need that (which I think we probably won't). WDYT?

Reply:
I think we can still keep the schema-version param as we should not define the version based on new entries, but the other way around tho.

By default the schema version will always point to the latest version, which is 1.1 in this case, unless someone want to fall back to 1.0, which they can with the parameter.

Reply:
Have offline discussion and will remove schema-version params, will keep 1.1 as the only default.

Thanks.